### PR TITLE
Feature: best program ever

### PR DESCRIPTION
**Pull request by edwinvautier**

### CODE REVIEW

File: helloworld.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""My hardest program ever"""
```
Comment (blyndusk) [inline helloworld.py]: `#`instead of `"""` ?

File: helloworld.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+def say_hello():
+  """print hello"""
```
Comment (blyndusk) [inline helloworld.py]: `#`instead of `"""`?

File: helloworld.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  print("Hello")
```
Comment (blyndusk) [inline helloworld.py]: `print('Hello World !')`

File: helloworld.py (hunk 1 above)
Comment (edwinvautier) [reply helloworld.py]: Docstring convention says that you should put triple `"`

**Pull request merged**